packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page/entity-editor/properties-section/property-table/cells/value-cell/value-cell-editor/array-editor/inline-text-editor.tsx

```diff
@@ -0,0 +1,28 @@
+import { TextField } from "@hashintel/hash-design-system";
+
+export const InlineTextEditor = ({
```

Comment:
What about numbers? It looks like the array editor is only doing strings at the moment

Reply:
Yes, it's only doing the strings at the moment, gonna support other types in follow-up PRs as I mentioned in the PR description

Reply:
Ah, okay – I thought by "support multiple types" you meant supporting arrays of mixed types (e.g. `(string | number)[]`